tests/e2e/constants/PLUGIN_TEST_CONSTANTS.ts

```diff
@@ -10,6 +10,9 @@
 export const PLUGIN_TEST_CONSTANTS: { TS_SAMPLE_LIST: string } = {
 	/**
 	 * dashboard samples to check in RecommendedExtensions.spec.ts
+	 * full list Node.js MongoDB,Node.js Express,Java 11 with Lombok,Java 11 with Quarkus,Python,.NET,C/C++,Go,PHP,Ansible
```

Comment:
this sentence looks redundant because is duplicating TS_SAMPLE_LIST constant value